components/TheFooter.tsx

```diff
@@ -1,64 +1,102 @@
+'use client';
+import { useState } from 'react';
 import enums from '@/enums/OFYears/ImageUrl2024';
 import React from 'react';
 import Image from 'next/image';
 import Link from 'next/link';
 import Routes from '@/enums/routes';
-import { FaFacebook, FaTwitter, FaInstagram, FaEnvelope } from 'react-icons/fa';
+import {
+    RiFacebookCircleFill,
+    RiTwitterFill,
+    RiInstagramFill,
+} from 'react-icons/ri';
+import {
+    RiFacebookCircleLine,
+    RiTwitterLine,
+    RiInstagramLine,
+} from 'react-icons/ri';
+
 import ExternalLink from '@/enums/externalLinks';
 
 const TheFooter = () => {
+    const [isFBHovered, setIsFBHovered] = useState(false);
+    const [isTwtHovered, setIsTwtHovered] = useState(false);
+    const [isIGHovered, setIsIGHovered] = useState(false);
```

Comment:
@Loweso you can replace this by using tailwind's peer modifier:

https://tailwindcss.com/docs/hover-focus-and-other-states